Definitions/ExperimentType.m

```diff
@@ -4,6 +4,7 @@
         LightIntensity = 2;
         GratingOrientation = 3;
         RandomDots = 4;
+        SoundDiscrimination = 5;
```

Comment:
How about we call it: `AuditoryLevelDiscrimination`?, and we can also call internally in the code any experimentx-related variable to `AudLvl`

Reply:
What about SoundLevelDiscrimination?

Reply:
> What about SoundLevelDiscrimination?

Hmm, I'm really not sure whether audio or sound is more appropriate here. I'm not a native speaker so I don't have a good sense for it as well, a quick google search shows [this](https://www.lizhardwick.co.uk/blog/audio-or-sound#:~:text=v%20Sound...-,Audio%20v%20Sound,from%20any%20format%20or%20instrument.), [this](https://english.stackexchange.com/questions/8420/differences-between-audio-and-sound) and [that](https://www.quora.com/What-are-the-differences-between-audio-and-sound). That makes me more leaning towards audio, but again my google search results might be biased.
Do you know whether someone in the literature has done a similar task? If so, then we can adapt their name. We can also check with Nassi and Mostafa if they have more knowledge about the topic.